fcd_purchase_order/models/purchase_order.py

```diff
@@ -46,7 +46,16 @@ def _purchase_order_add_line_open_wizard(self, purchase_id):
             'type': 'ir.actions.act_window',
         }
 
+    def button_confirm(self):
+        res = super(PurchaseOrder, self).button_confirm()
+        self.create_fcd_documents()
```

Comment:
button_confirm is multi (see https://github.com/odoo/odoo/blob/cc0060e889603eb2e47fa44a8a22a70d7d784185/addons/purchase/models/purchase.py#L405)

Suggested code:
```suggestion
        for order in self:
             order.create_fcd_documents()
```